azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -262,6 +262,9 @@ public void run() throws Exception {
       }
     }
 
+    // Append any JVM args needed for job commands
```

Comment:
Please rename appendJVMArgs. Also comments wherever you have mentioned JVM args.

Reply:
Ack